src/CRTM_Module.F90

```diff
@@ -35,7 +35,8 @@ MODULE CRTM_Module
   USE CRTM_K_Matrix_Module
   ! ...The aerosol optical depth tool
   USE CRTM_AOD_Module
-
+  ! ...Planck Functions
+  USE CRTM_Planck_Functions
```

Comment:
Hi @erinjones2. So, you don't have the option to `USE CRTM_Planck_Functions` directly in the GSI?

Reply:
Hi @StegmannJCSDA: at least using 2.3.0, the GSI will not compile with my changes without `USE CRTM_Planck_Functions` added to CRTM_Module.fpp.

@BenjaminTJohnson: Thanks for the note. I'll see if I can modify this to make the PR against release/REL-2.4.0_emc (it may also be needed for release/REL-2.4.0).

Reply:
Sounds good.